JitsiConference.js

```diff
@@ -3327,3 +3313,43 @@ JitsiConference.prototype.setE2EEKey = function(key) {
 
     this._e2eeCtx.setKey(key);
 };
+
+/**
+ * Setup E2EE for the sending side, if supported.
+ * Note that this is only done for the JVB Peer Connecction.
+ *
+ * @returns {void}
+ */
+JitsiConference.prototype._setupSenderE2EEForTrack = function(track) {
+    const jvbPc = this.jvbJingleSession ? this.jvbJingleSession.peerconnection : null;
+
+    if (!this.isP2PActive() && jvbPc && this._e2eeCtx) {
```

Comment:
do you really  need the `isP2PActive()` check?

Reply:
I suppose you don't because you're working with the JVB peerconnection anyway

Reply:
Right, I could simplify!